### PR TITLE
fix bug of q5_k_s w/ imatrix

### PR DESCRIPTION
**Pull request by n1ck-guo**
Test list:

| model                      | format | accuracy | infer |
| -------------------------- | ------ | -------- | ----- |
| Qwen2.5-3B-Instruct        | q5_k_s | 0.6389   | ✓     |
| falcon-three-7b            | q5_k_s | 0.6645   | ✓     |
| Llama-3.2-3B               | q5_k_s | 0.6928   | ✓     |
| Meta-Llama-3.1-8B-Instruct | q5_k_s | 0.7242   | ✓     |
| Meta-Llama-3.1-8B-Instruct | q2_k_s | 0.6511   | ✓     |
| Meta-Llama-3.1-8B-Instruct | q3_k_s | 0.7107   | ✓     |
| Meta-Llama-3.1-8B-Instruct | q6_k_s | 0.7287   | ✓     |
| Qwen2.5-7B-Instruct        | q2_k_s | 0.6975   | ✓     |
| Qwen2.5-7B-Instruct        | q4_k_s | 0.6928   | ✓     |
| Qwen2.5-7B-Instruct        | q3_k_s | 0.6771   | ✓     |
| Qwen2.5-7B-Instruct        | q6_k_s | 0.6930   | ✓     |
| Qwen2.5-7B-Instruct        | q5_k_s | 0.6749   | ✓     |

### CODE REVIEW

File: auto_round/export/export_to_gguf/packing.py [hunk 1]
```diff
@@ -652,8 +647,7 @@ def q6_k_quant_block(blocks: np.array, scale=None, zp=None, wmin_m=None, d_scale
         scales = scale.reshape(-1, QK_K // 16)
         output_d = d_scale.reshape(-1, 1).to(torch.float32)
         iscales = torch.where(output_d == 0, 0, 1 / output_d)
-        inverse_scale = torch.where(scales == 0, 0, 1.0 / scales)
-        all_L = (torch.round(blocks * inverse_scale).clip(-32, 31) + 32).to(torch.uint8)
+        all_L = torch.zeros_like(blocks, dtype=torch.uint8)
```
Comment (wenhuach21) [inline auto_round/export/export_to_gguf/packing.py]: for sym, the init value is not zero

**Review by wenhuach21 — APPROVED** [review]
